src/tests/parser_test.py

```diff
@@ -562,20 +563,70 @@ def test_matrix_multiplication_with_decorators(code):
     assert inner_function.decorators[0].name == 'a'
 
 
-def test_module_publicity():
-    """Test that a module that has a single leading underscore is private."""
+@pytest.mark.parametrize("parent_path", (
+    Path("package") / "another_package",
+    Path("")
+))
+def test_module_publicity(parent_path):
```

Comment:
I think it's also worth parametrizing all the examples below. That will significantly improve the readability of this test code and will make sure all of them are run independently.

You can modify the mark to hold file path and the expected publicity and simply check that below instead of manually repeating all the parse calls and asserts :)

Reply:
Good suggestion! I will re-work it, and notify when finished. :)

Reply:
@samj1912 Let me know if this addresses your comments https://github.com/PyCQA/pydocstyle/pull/494/commits/cec57934bf781f3d4956bdcc2c8c62bd6123cc73.

> You can modify the `mark.parametrize` to hold the file path **and the expected publicity**.

If we want a single `test_module_publicity` instead of two separate ones for **public** and **private**, then I'm happy to refactor that as well.